### PR TITLE
Use compose project name

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

- removes hardcoded container names
- removes hardcoded network - network is now configurable via env variable
- updates nginx config to take custom api host and port

Can only be used with [the updated version of the upgrade service](https://github.com/medic/cht-upgrade-service/tree/16-docker-compose-project-name_ )

medic/cht-core#7802

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by dianabarsan** [comment]
Hey @latin-panda 
This is the companion code PR for the cht-upgrade-service one. Please evaluate together. Thanks a lot!

File: tests/page-objects/standard/enketo/pregnancy.wdio.page.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const getFollowUpSMS = async () => {
 };
 
 const submitPregnancy = async () => {
+  await browser.takeScreenshot();
```
Comment (latin-panda) [inline tests/page-objects/standard/enketo/pregnancy.wdio.page.js:60]: No sure if this is intended to stay or temporary

**Review by latin-panda — APPROVED** [review]
LGTM, I ran the e2e locally and the pregnancy e2e failed, I wonder if it's the same error you saw. 
Screen Shot 2022-10-12 at 4 59 17 pm

File: tests/page-objects/default/contacts/contacts.wdio.page.js [hunk 2]
```diff
@@ -228,6 +228,7 @@ const openForm = async (name) => {
   const parent = await newActionContactButton().parentElement();
   await browser.waitUntil(async () => await parent.getAttribute('aria-expanded') === 'true');
 
+  await browser.takeScreenshot();
```
[the remark this answers is not in the document]
Comment (dianabarsan) [reply tests/page-objects/default/contacts/contacts.wdio.page.js:231]: oh, temporary. thanks for noticing! I was debugging this e2e test :)

**Comment by dianabarsan** [comment]
hey @Hareet 
This is the PR that adds the nginx template so we pass the api host as an environment variable.

**Pull request merged**